fission/src/systems/scene/SceneRenderer.ts

```diff
@@ -137,6 +161,21 @@ class SceneRenderer extends WorldSystem {
             gradientMap: gradientMap,
         })
     }
+
+    /* 
+     * Updates the skybox colors based on the current theme
+
+     * @param currentTheme: current theme from ThemeContext.useTheme()
```

Comment:
Probably best to have this than not. I would prefer that we standardize whether or not we want these kinds of doc strings at some point but that's a moot point anyways.

Looks good to me!